keps/693-multikueue/README.md

```diff
@@ -0,0 +1,234 @@
+# KEP-693: MultiKueue
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+Introduce an new AdmissionCheck (called MultiKueue) with dedicated API
+and controller that will provide multi-cluster capabilities to Kueue.
+
+## Motivation
+Many of Kueue's users are running multiple clusters and would like to
+have a way to easily distribute batch jobs across them to keep all of
+them utilized. Without a global distribution point, some clusters may
+get less jobs they are able to process, while the others get more, leading
+to underutilization and higher costs. 
+
+### Goals
+* Allow Kueue distribute batch jobs across multiple clusters,
+while maintaining the specified quota limits.
+* Provide users with a single entry point through which the jobs
+can be submitted and monitored, just like they were running in
+a single cluster.
+* Be compatible with all Kueue's features (priorities, borrowing, preemptions, etc)
```

Comment:
How do you plan to do preemption? By leveraging `kube-scheduler` in worker clusters for preemption or will Kueue handle it by itself across all worker clusters?

Reply:
Kueue has its own preemption mechanism that works on the job level (preempts the whole thing) instead of pod level (preempts only a fraction of job).